_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -52,8 +54,9 @@ test.describe('Popular Repositories', () => {
     });
 
     await test.step('Apply filter and clear it', async () => {
-      const searchInput = page.getByRole('textbox', { name: 'Filter by name/url' });
-      await searchInput.fill('EPEL 8 Everything x86_64');
+      await page
+        .getByRole('textbox', { name: 'Filter by name/url' })
+        .fill('EPEL 8 Everything x86_64');
```

Comment:
the string/name `'EPEL 8 Everything x86_64'` is used in several places in this file, why not use a variable instead? like for the EPEL 9 repo 🤔

Reply:
Fixed